### PR TITLE
curl, docker, git, git-add, git-commit, git-status, npm: add Indonesian translation

### PR DESCRIPTION
**Pull request by irhamdz**
- [x] The page (if new), does not already exist in the repo.

### CODE REVIEW

File: pages.id/common/curl.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- Unduh konten URL ke file:
+
+`curl {{http://example.com}} -o {{filename}}`
+
+- Unduh file, simpan hasilnya dengan nama file yang ditentukan oleh URL:
+
+`curl -O {{http://example.com/filename}}`
+
+- Unduh file, mengikuti pengalihan [L] okasi, dan secara otomatis [C] melanjutkan (melanjutkan) transfer file sebelumnya:
```
Comment (navarroaxel) [inline pages.id/common/curl.md:15]:
```suggestion
- Unduh file, mengikuti pengalihan [L] okasi, dan secara otomatis [C] melanjutkan transfer file sebelumnya:
```

File: pages.id/common/curl.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# curl
+
+> Mentransfer data dari atau ke server.
+> Mendukung sebagian besar protokol, termasuk HTTP, FTP, dan POP3.
+> Informasi selengkapnya: <https://curl.haxx.se>.
+
+- Unduh konten URL ke file:
+
+`curl {{http://example.com}} -o {{filename}}`
```
Comment (navarroaxel) [inline pages.id/common/curl.md:9]: Could you translate the arguments like `filename` to be easier for non-English speakers?

File: pages.id/common/git-add.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# git add
+
+> Menambahkan file yang diubah ke indeks.
+> Informasi selengkapnya: <https://git-scm.com/docs/git-add>.
+
+- Tambahkan file ke indeks:
+
+`git add {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.id/common/git-add.md:8]: Could you translate `path/to/file`?

File: pages.id/common/git-commit.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# git commit
+
+> Komit file ke dalam sebuah repositori.
+> Informasi selengkapnya: <https://git-scm.com/docs/git-commit>.
+
+- Komit file bertahap ke repositori dengan sebuah pesan:
+
+`git commit -m {{message}}`
+
+- otomatis merubah semua file yang dimodifikasi menjadi ke status stage dan menambahkan sebuah pesan:
```
Comment (navarroaxel) [inline pages.id/common/git-commit.md:10]:
```suggestion
- Otomatis merubah semua file yang dimodifikasi menjadi ke status stage dan menambahkan sebuah pesan:
```

File: pages.id/common/git-commit.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`git commit -a -m {{message}}`
+
+- Ganti komit terakhir dengan perubahan yang ada di status stage saat ini:
+
+`git commit --amend`
+
+- komit file tertentu (yang sudah di status stage):
```
Comment (navarroaxel) [inline pages.id/common/git-commit.md:18]:
```suggestion
- Komit file tertentu (yang sudah di status stage):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @irhamdz, welcome and thanks for your submission. I left some comments.

**Comment by navarroaxel** [comment]
@irhamdz the commit author should match with your GitHub user to pass the license/cla check. Do you need some help fixing the commit author?

**Comment by irhamdz** [comment]
> @irhamdz the commit author should match with your GitHub user to pass the license/cla check. Do you need some help fixing the commit author?

yes please I still confuse how to fix it

**Comment by navarroaxel** [comment]
@irhamdz You should amend your commit using:
```bash
git commit --amend --author 'Your name <user@example.com>'
git push --force
```
And the name and email must match your GitHub account.

**Comment by irhamdz** [comment]
@navarroaxel I update the PR and thanks for helping me with change the author commit :)

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**